soc/nxp/imxrt/imxrt11xx/Kconfig

```diff
@@ -45,6 +45,7 @@ config SOC_SERIES_IMXRT11XX
 	select HAS_MCUX_XBARA
 	select HAS_SWO
 	select HAS_PM
+	select HAS_MCUX_CSI
```

Comment:
Why do we need to select this KConfig? These Kconfigs have largely been replaced with `DT_HAS_XXX` Kconfigs, and are legacy from before that feature existed